tests/test_providers.py

```diff
@@ -90,6 +92,45 @@ def test_recordid_provider(app, db):
         pytest.raises(AssertionError, RecordIdProvider.create, pid_value='3')
 
 
+def test_recordid_provider_v2(app, db):
```

Comment:
Minor: It's not checking the split got done appropriately, e.g.:

``` python
parts = pid_value.split('-')
assert len(parts) == 2
assert len(parts[0]) == 5
assert len(parts[1] == 5)
```

Reply:
That's a nicer test. Thanks.